dbicdh/PostgreSQL/upgrade/69-70/001-auto.sql

```diff
@@ -1,5 +1,11 @@
 -- Convert schema '/hdd/openqa-devel/repos/openQA/script/../dbicdh/_source/deploy/69/001-auto.yml' to '/hdd/openqa-devel/repos/openQA/script/../dbicdh/_source/deploy/70/001-auto.yml':;
 
 ;
--- No differences found;
+BEGIN;
 
+;
+ALTER TABLE job_modules ADD COLUMN always_rollback integer DEFAULT 0 NOT NULL;
```

Comment:
this doesn't work - you need a new schema version

Reply:
Unfortunately, I was able only to find the documentation to introduce changes to the DB this way. Do you think @Martchus can give me a hand here?

Reply:
I can, but I usually just go through these steps myself: https://github.com/os-autoinst/openQA/blob/master/docs/Contributing.asciidoc#managing-the-database

You forgot step 1. here (updating the schema version number) and instead overwrote the previous migration files.

Reply:
Ah, I see. Will fix that. Thanks a lot!